CHANGELOG.md

```diff
@@ -98,6 +98,7 @@ Here is an overview of all new **experimental** features:
 
 - **General**: Scalers cache uses a mutex to prevent concurrent actions ([#6273](https://github.com/kedacore/keda/issues/6273))
 - **AWS CloudWatch Scaler**: Properly parse multiple values of `dimensionName` & `dimensionValue` lists ([#6236](https://github.com/kedacore/keda/issues/6236))
+- **AWS Scalers**: Add AWS region to the AWS Config Cache key ([#6128](https://github.com/kedacore/keda/issues/6128))
```

Comment:
2.16 is already released, we should move this to unreleased

Reply:
done